website/docs/d/instance.html.markdown

```diff
@@ -54,6 +54,7 @@ are exported:
 ~> **NOTE:** Some values are not always set and may not be available for
 interpolation.
 
+* `ami` - The AMI in use on the Instance.
```

Comment:
Hey @automaticgiant 

Do you mind updating this to `The ID of the AMI used to launch the instance.`, which is the description extracted from the API? (https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Instance.html under  `imageID`)

Thanks!